en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -157,12 +163,58 @@ configured in the previous step:
     </tr>
     </table>
 
-12. Save the output mapping.
+8.  Click **Generate** to generate output mappings automatically.
+
+    !!! Tip
+        Alternatively, you can manually add the mappings:
+        1. Click **Add New** to open the **Add Output Mapping** page.
+        2. Enter the following output element details.
+            <table>
+            <tr>
+                    <th>Property</th>
+                    <th>Description</th>
+                </tr>
+            <tbody>
+            <tr class="odd">
+            <td>Datasource Type</td>
+            <td>column</td>
+            </tr>
+            <tr class="even">
+            <td>Output Field Name</td>
+            <td>EmployeeNumber</td>
+            </tr>
+            <tr class="odd">
+            <td>Datasource Column Name</td>
+            <td>EmployeeNumber</td>
+            </tr>
+            <tr class="even">
+            <td>Schema Type</td>
+            <td>String</td>
+            </tr>
+            </tbody>
+            </table>   
+        3.  Save the element.
+        4.  Follow the same steps to create the following output elements:
+
+            | Datasource Type | Output Field Name | Datasource Column Name | Schema Type |
+            |-----------------|-------------------|------------------------|-------------|
+            | column          | FirstName         | FirstName              | string      |
+            | column          | LastName          | LastName               | string      |
+            | column          | Email             | Email                  | string      |
+   
+ 
+9.  Click **Save** to save the query.
+    ![](../../assets/img/tutorials/data_services/output_mapings.png)
+
+##### Creating a resource to invoke the query
 
-13. Right-click the output mapping and go to **Add Output Mapping → Add
-    Element** to create an element.
+Now, let's create a REST resource that can be used to invoke the query.
 
-14. Enter the following element details.
+1.  Click **Resources** to expand the section. 
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
+2.  Click **Add New** to open the **Create Resource** page.
+    ![](../../assets/img/tutorials/data_services/create_resource.png)
```

Comment:
@DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.